### ISSUE TITLE
Notification that you've been assigned a review

### ISSUE THREAD

**Issue by lmkw** [NUARIG/competitions#136]
Should there be an email? What information should be in it? Should there be notifications when you log in? Where would they go?

**Comment by lmkw** [comment]
dependent on #147

**Comment by jseraf** [comment]
Duplicate of #144. Un-/assign email notifications added there. 

Note: Text of outgoing email needs to be re-written. See:
https://github.com/NUARIG/competitions/tree/develop/app/views/reviewer_mailer
